### PR TITLE
Addition of the commands accessible in shimming-toolbox in the README.rst

### LINKED ISSUE #274: Uniformity of the README and help of the commands
In the README there isn't a list of all st_ commands and the example it links don't show all of them. It would be nice to add a list of all the commands accessible in the README.rst

Futhermore, the "st_mask -h" command give the different commands but not their parameters (ex: the box comand need -size -input -output but they aren't in the "st_mask -h")
The "st_prepare_fieldmap" doesn't have the [requiered] to the phase path input.

### PR DESCRIPTION
**Pull request by KevinBerard**
## Checklist

- [X] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied the relevant labels to this PR
- [x] I've added relevant tests for my contribution
- [x] I've updated the documentation and/or added correct docstrings
- [X] I've assigned a reviewer

## Description

Addition of the commands in the README.rst and a brieve description of the nested usage of the command help.

## Linked issues

FIxes #274

### CODE REVIEW

**Comment by po09i** [comment]
The build is failing, here are the info on how to build the doc locally https://github.com/shimming-toolbox/shimming-toolbox/wiki. If it builds locally, it "should" build online.

**Comment by KevinBerard** [comment]
> The build is failing, here are the info on how to build the doc locally https://github.com/shimming-toolbox/shimming-toolbox/wiki. If it builds locally, it "should" build online.

The modification as been done.

**Comment by po09i** [comment]
I have PR #277 on the way which adds the CLIs that were missing to the doc. Maybe just linking to the CLI page would be better instead of updating different places? Here is what the [CLI page ](https://shimming-toolbox-py--277.org.readthedocs.build/en/277/cli_reference/cli.html) would look like.

**Comment by KevinBerard** [comment]
I completly agree. I will wait for the PR #277 to be done to make sure the permanent link will be set in the README

**Comment by po09i** [comment]
#277 got merged, we can merge master in this branch then proceed with what was discussed

File: README.rst [hunk 1]
```diff
@@ -77,9 +77,12 @@ The primary way to use ``shimming-toolbox`` is through the command line. For exa
   -h, --help            Show this message and exit.
 
 .. admonition:: Note
-
+
```
Comment (po09i) [inline README.rst]:
```suggestion

```

**Review by po09i — APPROVED** [review]
Looks good!

**Pull request merged**